diff --git a/dist/index.d.ts b/dist/index.d.ts
index f50323f6c64d4e1b06b76aeb1699692f8d21f323..856f79cf5b64f33296049ac263b22e0b938037cb 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -1056,10 +1056,11 @@ declare function resolvePreset<Theme extends object = object>(presetInput: Prese
 declare function resolvePresets<Theme extends object = object>(preset: PresetOrFactoryAwaitable<Theme>): Promise<Preset<Theme>[]>;
 declare function resolveConfig<Theme extends object = object>(userConfig?: UserConfig<Theme>, defaults?: UserConfigDefaults<Theme>): Promise<ResolvedConfig<Theme>>;
 /**
- * Merge multiple configs into one, later ones have higher priority
+ * Merge multiple configs into one, later ones have higher priority PresetOrFactoryAwaitable
  */
 declare function mergeConfigs<Theme extends object = object>(configs: UserConfig<Theme>[]): UserConfig<Theme>;
 declare function definePreset<Options extends object | undefined = undefined, Theme extends object = object>(preset: PresetFactory<Theme, Options>): PresetFactory<Theme, Options>;
+declare function definePreset<Options extends object | undefined = undefined, Theme extends object = object>(preset: PresetFactoryAwaitable<Theme, Options>): PresetFactoryAwaitable<Theme, Options>;
 declare function definePreset<Theme extends object = object>(preset: Preset<Theme>): Preset<Theme>;
 
 declare const defaultSplitRE: RegExp;
